drivers/ieee802154/Kconfig

```diff
@@ -39,6 +39,13 @@ config IEEE802154_RDEV
 	help
 	  PHY is a ranging-capable device (RDEV)
 
+config IEEE802154_VENDOR_OUI
+	int "Vendor Organizationally Unique Identifier"
+	default 0
```

Comment:
We cannot use any default value here. I would rather change this to boolean and add another one like IEEE802154_VENDOR_OUI_VALUE without default value to ensure that user will have to fill this option when IEEE802154_VENDOR_OUI is enabled.